lib/src/table/table-td/table-td.component.html

```diff
@@ -0,0 +1,6 @@
+<span *ngIf="valueType === 'string'">{{ row }}</span>
+<span *ngIf="valueType === 'number'">{{ row }}</span>
+<span *ngIf="valueType === 'datetime'">{{ row | date: "yyyy-MM-dd HH:mm:ss" }}</span>
+<span *ngIf="valueType === 'date'">{{ row | date: "yyyy-MM-dd" }}</span>
+<span *ngIf="!valueType">–</span>
+<span *ngIf="!row">–</span>
```

Comment:
switch case would be nice to have heere
```suggestion
<ng-container [ngSwitch]="valueType">
  <span *ngSwitchCase="'string'">{{ row }}</span>
  <span *ngSwitchCase="'number'">{{ row }}</span>
  <span *ngSwitchCase="'datetime'">{{ row | date: "yyyy-MM-dd HH:mm:ss" }}</span>
  <span *ngSwitchCase="'date'">{{ row | date: "yyyy-MM-dd" }}</span>
  <span *ngSwitchDefault>–</span>
</ng-container>
```

Reply:
that's a good suggestion, thanks :)

Reply:
one thing that i forgot here is the default value doesnt need to have "-" all the time, it can be empty, i dont know why you chose to add that character, i suggest to remove it and keep the span blank, what do you think